### PR TITLE
Add method to retrieve server plugins from Python API

### PR DESCRIPTION
**Pull request by pblottiere**

### CODE REVIEW

**Comment by rldhont** [comment]
ping @Gustry

File: tests/src/server/CMakeLists.txt [hunk 1]
```diff
@@ -15,6 +15,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}
 # Tests:
 
 set(TESTS
+  test_qgsserver_interface.cpp
```
Comment (elpaso) [inline tests/src/server/CMakeLists.txt:18]:
```suggestion
  testqgsserverinterface.cpp
```

I think the naming convention for cpp tests has no `_`

**Review by elpaso — APPROVED** [review]

File: tests/src/server/CMakeLists.txt (hunk 1 above)
Comment (pblottiere) [reply tests/src/server/CMakeLists.txt]: Indeed you're right :+1:

**Comment by Gustry** [comment]
Nice addition.

Just for information, it was already possible before using this import from Python, in case you need it before 3.30
```python
from qgis.utils import server_active_plugins
```

**Review by rldhont — APPROVED** [review]

**Comment by rldhont** [comment]
@pblottiere I think you can merge, isn't it ?

**Comment by rldhont** [comment]
@pblottiere I think, you have to rebase. I can't merge.

**Comment by rldhont** [comment]
Waiting for rebase

**Pull request closed without merging**